index.html

```diff
@@ -11,11 +11,13 @@
 
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
     <link rel="stylesheet" href="bower_components/primer-css/css/primer.css">
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/plyr/dist/plyr.css">
     <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" id= "cssselector" href="css/style1.css">
```

Comment:
probably should change the `id` to something more descriptive like `currentTheme`